test/integration/capacity.test.ts

```diff
@@ -58,12 +63,34 @@ describe('capacity', () => {
       expect(response.status).to.equal(200)
       expect(response.body.length).to.equal(1)
       expect(response.body[0]).to.deep.equal({
-        id: capacityId,
+        id: seededCapacityId,
         owner: selfAlias,
-        status: DemandStatus.Created,
+        state: DemandState.created,
         parametersAttachmentId,
       })
     })
+
+    test('it should create a capacity on-chain', async () => {
```

Comment:
I think we need a bit more test coverage. What happens if the `runProcess` call fails?

Reply:
is this enough, `500` when `run-process` throws an error https://github.com/digicatapult/dscp-matchmaker-api/pull/17/commits/5ea1e024b366e1a286a3243e13427e81921fe576?